.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = extensions/BounceHandler
 	url = https://github.com/wikimedia/mediawiki-extensions-BounceHandler.git
 	branch = REL1_31
+[submodule "extensions/Nuke"]
+	path = extensions/Nuke
+	url = git@github.com:wikimedia/mediawiki-extensions-Nuke.git
```

Comment:
Maybe we should use the https-URLs here as well. Nitpicking, I know. :-)

Reply:
Oops, fixed that...